Gemfile

```diff
@@ -60,7 +60,7 @@ gem 'rest-client'
 gem 'retries'
 gem 'ruby-prof'
 gem 'rubyzip'
-gem 'turbo-rails', '~> 0.5.9'
+gem 'turbo-rails', '~> 0.7.13'
```

Comment:
is this upgrade necessary for this change?   31 files changed is a lot for a new feature.

Reply:
I made this change when I encountered this bug https://github.com/hotwired/turbo/issues/396.  While this didn't fix the bug I was looking at (I worked around it), this is how Andrew tested it, so to be respectful of his time, I'd rather not separate this change and ask him to test again.